ArduCopter/mode_planckrtb.cpp

```diff
@@ -22,6 +22,10 @@ bool ModePlanckRTB::init(bool ignore_checks){
 }
 
 void ModePlanckRTB::run(){
+    if(_is_landing && copter.planck_interface.was_last_request_rejected() && (copter.planck_interface.get_last_cmd_req_id() == PLANCK_CMD_REQ_LAND)) {
```

Comment:
Why not just update `_is_landing` when the REQ_LAND ACK comes back?

Reply:
It gets tricky, right? Has to land when ready, but also only send the land req once, or it will keep initializing ModePlanckLand::init() and keep getting acks, and ruin the ack status. I'll give it a try though.